polus-csv-statistics-plugin/plugin.json

```diff
@@ -25,7 +25,9 @@
                 { "description": "Kurtosis", "enum": [ "kurt" ] },
                 { "description": "Total Count", "enum": [ "count" ] },
                 { "description": "Maximum Value", "enum": [ "max" ] },
-                { "description": "Minimum Value", "enum": [ "min" ] }
+                { "description": "Minimum Value", "enum": [ "min" ] },
+                {"description": "Inter Quartile Range", "enum": [ "iqr" ]},
+                { "description": "All parameters","enum": [ "mean","median","std","var","skew","kurt","count","max","min","iqr"]}
```

Comment:
enum cannot be a list with more than one value. This must be a single value, such as "all". This means you will also have to change your Python code to handle the input.